src/hero.tsx

```diff
@@ -147,11 +166,13 @@ const Hero = React.forwardRef<HTMLDivElement, HeroProps>(
                                 [styles.vars.height]: height ?? '100%',
                             }),
                         }}
-                        className={classnames(styles.container, styles.containerMobile)}
+                        className={classnames(styles.container, styles.containerMobile, {
+                            [styles.containerMinHeight]: !noPaddingY,
+                        })}
                     >
                         {media}
                         <Layout isInverse={isInverse}>
-                            <Box paddingTop={24} paddingBottom={isInsideSlideShow ? 48 : 24}>
+                            <Box paddingTop={24} paddingBottom={noPaddingY ? 0 : isInsideSlideShow ? 48 : 24}>
```

Comment:
if I understand this correctly, if I use noPaddingY inside a slideshow, I would need to add the pageBullets outside (or reserve this space inside the "slide", correct? It is okay, but just to make sure.

Reply:
Yes, that is correct :)

Reply:
Actually, giving it a second thought: I believe it's okay to add the bottom padding in case we put the Hero inside a slideshow (and set it to 0 otherwise if the noPaddingY prop is true). I've fixed this